pyproject.toml

```
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "dj-registry"
+version = "0.1.0"
```

The version of the current release is 0.2.0 ([PyPI](https://pypi.org/project/dj-registry/), [GitHub tag](https://github.com/yychen/dj-registry/releases/tag/0.2)). The version assigned here should be `0.2.1` (since we do not expect any new features before the next release). Also, check out the term [`Semantic Versioning`](https://semver.org/) to learn when to increment the version numbers.
```suggestion
version = "0.2.1"
```